harness/determined/cli/config_policies.py

```diff
@@ -0,0 +1,158 @@
+    elif args.workload_type.upper() == "NTSC":
+        workload_type = "NTSC"
+    else:
+        raise cli.errors.CliError(
+            "Failed to delete config policies: Invalid workload type provided."
+            + " Valid options: 'experiment' and 'ntsc'"
+        )
+
+    wksp = api.workspace_by_name(sess, args.workspace_name)
+    bindings.delete_DeleteWorkspaceConfigPolicies(
+        sess, workloadType=workload_type, workspaceId=wksp.id
+    )
+    print(
+        f"Successfully deleted {workload_type} config policies for workspace {args.workspace_name}."
+    )
+    return None
+
+
+args_description: cli.ArgsDescription = [
+    cli.Cmd(
+        "config-policies",
+        None,
+        "manage config policies",
+        [
+            cli.Cmd(
+                "describe",
+                describe_config_policies,
+                "describe config policies",
+                [
+                    cli.Arg(
+                        "--workspace-name",
+                        type=str,
+                        required=True,  # change to false when adding --global
+                        help="config policies for the given workspace",
+                    ),
+                    cli.Arg(
+                        "--workload-type",
+                        type=str,
+                        required=True,
+                        help="config policies for the given workspace",
+                    ),
+                    cli.Group(cli.output_format_args["json"], cli.output_format_args["yaml"]),
+                ],
+                is_default=True,
+            ),
+            cli.Cmd(
+                "set",
+                set_config_policies,
+                "set config policies",
+                [
+                    cli.Arg(
+                        "--workspace-name",
+                        type=str,
+                        required=True,  # change to false when adding --global
+                        help="config policies for the given workspace",
+                    ),
+                    cli.Arg(
+                        "--workload-type",
+                        type=str,
+                        required=True,
+                        help="config policies for a given workload, options: Experiment and NTSC",
+                    ),
+                    cli.Arg(
+                        "--path",
+                        type=str,
+                        required=True,
+                        help="config policies for the given workspace",
+                    ),
+                ],
+            ),
+            cli.Cmd(
+                "delete",
+                delete_config_policies,
+                "delete config policies",
+                [
+                    cli.Arg(
+                        "--workspace-name",
+                        type=str,
+                        required=True,  # change to false when adding --global
+                        help="config policies for the given workspace",
```

Comment:
```suggestion
                        help="the workspace whose config policies are being deleted ",
```

Reply:
I actually prefer `config policies for the given workspace` but I think either is fine.

Reply:
Shouldn't the argument's help description correspond to what it is though?

Reply:
i'd say since there is a separate help string for the command itself, we don't have to customize the help strings based on which command it is in. But maybe @azhou-determined  can weigh in